examples/tutorials/basics/polygons.py

```diff
@@ -0,0 +1,96 @@
+"""
+Plotting polygons
+=================
+
+Plotting polygons is handled by the :meth:`pygmt.Figure.plot` method.
+
+This tutorial focuses on input data given as NumPy arrays. Besides NumPy arrays,
+array-like objects are supported. For plotting a :class:`geopandas.GeoDataFrame` object
+with polygon geometries, e.g. to create a choropleth map, see the gallery example
+:doc:`Choropleth map </gallery/maps/choropleth_map>`.
+"""
+
+# %%
+import numpy as np
+import pygmt
+
+# %%
+# Plot polygons
+# -------------
+#
+# Set up sample data points as NumPy arrays for the x and y values.
```

Comment:
Maybe we should add a sentence like: 

> A polygon is a closed shape defined by a series of data points with x and y coordinates, connected by line segments, with the start and end data points identical.

Reply:
Yeah, when reading your issue, I needed a moment to understand what you mean with "polygon" So adding such an explanation makes sense. Included your formulation slightly modified to the introduction part in commit https://github.com/GenericMappingTools/pygmt/pull/3593/commits/1ae944549812db1d32604b3436c92e0669c2a3d8.